types/aws-lambda/index.d.ts

```diff
@@ -30,6 +30,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
+/// <reference types="node" />
```

Comment:
Ah yes, this would be needed for `Buffer`. I'm surprised we didn't have this in here already, but it does make sense.